### ISSUE TITLE
BetaFish: keyboard doesn't work

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#185]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/betafish/

Can't access play/pause and reset with keyboard

(technically you can with one key shortcut, but you need to be able to hover over it with a mouse to discover that)

Recommendation:

Change divs to buttons

**Comment by alrra** [comment]
Closing in favor of #260.
